### ISSUE TITLE
400 error occurred without any change on my side 

### ISSUE THREAD

**Issue by jsifalda** [EasyChris/obsidian-to-notion#51]
hello, I am getting 400 error when trying to publish to notion.so... the secret and options are set correctly, as it was working before... any ideas?

<img width="313" alt="Screenshot 2023-06-19 at 12 15 27" src="https://github.com/EasyChris/obsidian-to-notion/assets/1549390/f69cd41c-57fd-46bc-9086-a6393734e755">
<img width="979" alt="Screenshot 2023-06-19 at 12 17 50" src="https://github.com/EasyChris/obsidian-to-notion/assets/1549390/4a89fccb-ab3f-4b65-bed0-d8c5a29c85d1">

**Comment by jsifalda** [comment]
at least, is there a way to debug this request? ability to see what error it returns? thx

**Comment by medibb** [comment]
I found a point:
when using a bullet list for the 4th level, I encountered a 400 error. However, the 3rd level integrated smoothly.

**Comment by jsifalda** [comment]
no, it does not work at all, not just simple line of text :(  (no bullet points included)

**Comment by razeranthom** [comment]
Not working.. even with simple notes (only text).

**Comment by haphanhp** [comment]
same thing happen for me
